core/src/main/java/org/mule/registry/MuleRegistryHelper.java

```diff
@@ -725,30 +712,59 @@ public Object applyProcessorsAndLifecycle(Object object) throws MuleException
     /**
      * {@inheritDoc}
      */
+    @Override
     public Object applyProcessors(Object object) throws MuleException
     {
-        return registry.getTransientRegistry().applyProcessors(object, null);
+        return inject(object);
     }
 
     /**
      * {@inheritDoc}
      */
+    @Override
+    @Deprecated
     public Object applyProcessors(Object object, int flags) throws MuleException
```

Comment:
it was a hack to manually bypass certain processors and certain lifecycle operations. Something that was never used and that we don't want anyway